lib/core/src/Cardano/Wallet/DB.hs

```diff
@@ -225,8 +225,13 @@ data DBLayer m s k = forall stm. (MonadIO stm, MonadFail stm) => DBLayer
     , rollbackTo
         :: PrimaryKey WalletId
         -> SlotId
-        -> ExceptT ErrNoSuchWallet stm ()
+        -> ExceptT ErrNoSuchWallet stm SlotId
         -- ^ Drops all checkpoints and transaction data after the given slot.
+        --
+        -- Returns the actual slot to which the database has rolled back. This
+        -- slot is guaranteed to be smaller or equal to the given point of
```

Comment:
> slot is guaranteed to be smaller or equal

Nit-pick: I'd find "earlier [slot]" or "before [slot]" easier to parse than "smaller [slot]".